src/logger/logger.ts

```diff
@@ -4,6 +4,13 @@ import * as uuid from 'uuid';
 import stringify from 'fast-safe-stringify';
 import isError from 'is-error';
 
+export const redactSecret = (data: any): any => {
```

Comment:
Also I think the return value is always `string` right?

Reply:
yes, changed